src/webgpu/shader/execution/expression/call/builtin/ceil.spec.ts

```diff
@@ -39,39 +38,27 @@ g.test('f32')
       .combine('vectorize', [undefined, 2, 3, 4] as const)
   )
   .fn(async t => {
-    const cfg: Config = t.params;
-    cfg.cmpFloats = correctlyRoundedMatch();
-
     const makeCase = (x: number): Case => {
-      return makeUnaryF32Case(x, Math.ceil);
+      return makeUnaryF32IntervalCase(x, ceilInterval);
```

Comment:
No, makeUnaryF32IntervalCase is responsible for quantizing to f32. I moved the quantization to the builders, since it needs to be done for every case for all of the test types, and made it explicit in the framework where/how the conversion is done.
This is also done right before calculating the acceptance interval, so we get rid of situations where we were actually calculating the interval for the non-quanitized input, which implementation will never actually see and getting subtle bugs.